dynamic-programming/coinChange.go

```diff
@@ -0,0 +1,29 @@
+/* Coin Change problem
+1. Finding the total number of combination that can be achieved with the given number of coins to get the specified sum.
```

Comment:
Please provide a more informative description. About the problem, how to solve it, and reference also.